resources/views/livewire/home/search.blade.php

```diff
@@ -0,0 +1,90 @@
+                                <figure class="{{ $user->is_company_verified ? 'rounded-md' : 'rounded-full' }} h-12 w-12 flex-shrink-0 overflow-hidden bg-slate-800 transition-opacity group-hover:opacity-90">
+                                    <img
+                                        class="{{ $user->is_company_verified ? 'rounded-md' : 'rounded-full' }} h-12 w-12"
+                                        src="{{ $user->avatar_url }}"
+                                        alt="{{ $user->username }}"
+                                    />
+                                </figure>
+                                <div class="flex flex-col overflow-hidden text-sm text-left">
+                                    <a
+                                        class="flex items-center space-x-2"
+                                        href="{{ route('profile.show', ['username' => $user->username]) }}"
+                                        wire:navigate
+                                        x-ref="parentLink"
+                                    >
+                                        <p class="text-wrap truncate font-medium dark:text-white text-black">
+                                            {{ $user->name }}
+                                        </p>
+
+                                        @if ($user->is_verified && $user->is_company_verified)
+                                            <x-icons.verified-company
+                                                :color="$user->right_color"
+                                                class="size-4"
+                                            />
+                                        @elseif ($user->is_verified)
+                                            <x-icons.verified
+                                                :color="$user->right_color"
+                                                class="size-4"
+                                            />
+                                        @endif
+                                    </a>
+                                    <p class="truncate text-slate-500 transition-colors group-hover:text-slate-400">
+                                        {{ '@'.$user->username }}
+                                    </p>
+                                </div>
+                                <x-follow-button
+                                    :id="$user->id"
+                                    :isFollower="auth()->check() && $user->is_follower"
+                                    :isFollowing="auth()->check() && $user->is_following"
+                                    class="ml-auto"
+                                    wire:key="follow-button-{{ $user->id }}"
+                                />
+                            </div>
```

Comment:
After changing the name & adding the markup for the user card component, utilise it here.
```suggestion
                        <li>
                            <x-user-card
                                data-parent=true
                                x-data="clickHandler"
                                x-on:click="handleNavigation($event)"
                                wire:key="user-{{ $result->id }}"
                                :user="$result"
                            />
                        </li>
```